Sources/SwifterSwift/AppKit/NSViewExtensions.swift

```diff
@@ -29,7 +23,7 @@ public extension NSView {
     @IBInspectable
     var borderWidth: CGFloat {
         get {
-            return layer?.borderWidth ?? 0
```

Comment:
Personally, I'm not a big fan of this enforcement ... I think although the language allows this, I think we should use in very specific case(personally I would use it more in cases where we could do one-liner e.g. var x { a + b } ). But I'll let you guys decide on that :) To me if we would enforce this I think we should enforce at least to try let the single expression in the same line e.g. `get { layer?.borderWidth ?? 0 }` I think it would be a good way to go

Reply:
See: https://github.com/nicklockwood/SwiftFormat/blob/master/Rules.md#redundantreturn

I think this is idiomatic Swift now and thus the default with SwiftFormat. Once the redundant returns are gone the remaining returns are clear indicators of early exit. I personally prefer this style, especially after working in other languages with implicit return (like Ruby).

Not a strong opinion, though. We can disable the `redundantReturn` rule if others agree. @guykogus? @omaralbeik? Chime in if you have opinions...

![image](https://user-images.githubusercontent.com/526195/89854908-fb187100-db49-11ea-8779-f3248893e7f0.png)

Reply:
Yeah, I like it very much but in a case by case basis, as I mentioned we should have a convention to use as we see fit, but not enforce as a rule everywhere :)

Reply:
I don't think we can customize it to that extent. Personally this doesn't bother me (because Ruby) but I don't have a strong opinion. Our convention is good even without autoformatting, I think. Let's turn off `redundantReturn`